From 8be40a86ee7a5802b228dfe3c1c81df1450540a3 Mon Sep 17 00:00:00 2001
From: Julia Reynolds <juliacr@google.com>
Date: Wed, 31 Aug 2022 12:58:41 -0400
Subject: [PATCH] Lower per-app notificationchannel limit

Test: PreferencesHelperTest
Bug: 240422263
Change-Id: I8c12e3fc73e4a88842af275feaf2acffcced0402
(cherry picked from commit f528b337dd48b7e8071269e07e610bd4a3668c75)
Merged-In: I8c12e3fc73e4a88842af275feaf2acffcced0402
(cherry picked from commit 36acdd675890296151b7a99e05fa7436d59a289b)
Merged-In: I8c12e3fc73e4a88842af275feaf2acffcced0402
---
 .../java/com/android/server/notification/PreferencesHelper.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/notification/PreferencesHelper.java b/services/core/java/com/android/server/notification/PreferencesHelper.java
index d5d5308015c7..c6cb3c0d2ab7 100644
--- a/services/core/java/com/android/server/notification/PreferencesHelper.java
+++ b/services/core/java/com/android/server/notification/PreferencesHelper.java
@@ -97,7 +97,7 @@ public class PreferencesHelper implements RankingConfig {
     private static final String NON_BLOCKABLE_CHANNEL_DELIM = ":";
 
     @VisibleForTesting
-    static final int NOTIFICATION_CHANNEL_COUNT_LIMIT = 50000;
+    static final int NOTIFICATION_CHANNEL_COUNT_LIMIT = 5000;
 
     private static final int NOTIFICATION_PREFERENCES_PULL_LIMIT = 1000;
     private static final int NOTIFICATION_CHANNEL_PULL_LIMIT = 2000;
-- 
2.38.1.273.g43a17bfeac-goog

